### ISSUE TITLE
epel-release ngnix task

### ISSUE THREAD

**Issue by hughnet** [bingoarun/ansible-elk#2]
Hi there,

There is an issue:
`TASK [ansible-elk : Installing EPEL repo] **************************************
fatal: [52.11.178.31]: FAILED! => {"changed": false, "failed": true, "msg": "No Package matching 'epel-release' found available, installed or updated", "rc": 0, "results": []}
    to retry, use: --limit @playbooks/cirrent_elk.retry`

You will need to add a couple things to make this work, shall I submit a PR? please let me know.

**Comment by bingoarun** [comment]
@hughnet : Yes please, BTW is it Redhat ?  (I have tested this only with Cent OS 7 - Openstack cloud image )
